### ISSUE TITLE
Variable Inclusion

### ISSUE THREAD

**Issue by lindechen** [jcochanc/AHS#4]
We should examine the variables that are correlated with the outcomes. Recall the outcomes our a categorical income bracket and education.

**Comment by CarolinaVelasco** [comment]
I compiled a list of variables that could potentially be used to build the quality index. I added this to the code, as well as put together a codebook as a csv file that has some annotations. Let me know what you think!

**Comment by CarolinaVelasco** [comment]
Actually, I'm not able to commit in source tree. This is how my screen looks like. Jerson, could you guide through the commit & push process? Thanks! 

<img width="1249" alt="Screen Shot 2019-04-11 at 10 44 47 PM" src="https://user-images.githubusercontent.com/31706702/56011077-aa6ddb00-5cab-11e9-9880-a34d0c2536fb.png">

**Comment by jcochanc** [comment]
Looks like you didn’t pull. Pull the updates then you should be able to push yours

**Comment by CarolinaVelasco** [comment]
I pulled, and then when I try to commit I get this: 
<img width="1164" alt="Screen Shot 2019-04-11 at 10 50 18 PM" src="https://user-images.githubusercontent.com/31706702/56011232-3da71080-5cac-11e9-9679-7439f572159d.png">

**Comment by jcochanc** [comment]
There’s a (1) by the pull for the branch suggesting you haven’t pulled everything.  Maybe it’s bugged. Hit pull and let me see what it says then

**Comment by CarolinaVelasco** [comment]
Yeah I've been pulling before trying to commit . This is what I get when I click on pull: 
<img width="1237" alt="Screen Shot 2019-04-11 at 10 53 12 PM" src="https://user-images.githubusercontent.com/31706702/56011321-b60dd180-5cac-11e9-8c45-cc68dfe46afc.png">

**Comment by jcochanc** [comment]
That’s good. Then hit okay. What happens after?

**Comment by CarolinaVelasco** [comment]
Apparently there are conflicts? 

<img width="1210" alt="Screen Shot 2019-04-11 at 10 55 04 PM" src="https://user-images.githubusercontent.com/31706702/56011349-ddfd3500-5cac-11e9-944e-1da25dc7e678.png">

**Comment by CarolinaVelasco** [comment]
I'm not sure what is the conflict, but these are the options: 
<img width="1229" alt="Screen Shot 2019-04-11 at 11 00 18 PM" src="https://user-images.githubusercontent.com/31706702/56011525-a5119000-5cad-11e9-83b6-40b2bf7589ab.png">

**Comment by jcochanc** [comment]
So it looks like you started writing in a file before you pulled everything. Otherwise this conflict wouldn’t have happened. Basically I may have pushed something and you had an old copy of the work. To fix this, follow the instructions on resolving merge conflicts. Essentially in one of the files you’ll see some parts that says HEAD and that’s why you have to resolve. Here’s a place to read about this: https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line

**Comment by jcochanc** [comment]
See, the things highlighted in green are the stuff you didn’t pull before starting your work. Just go into the file and delete that code if you don’t want it.

**Comment by CarolinaVelasco** [comment]
Okay, I solved the conflict. Now, I'm getting an error to pull because the file is too large: 
<img width="1127" alt="Screen Shot 2019-04-11 at 11 33 21 PM" src="https://user-images.githubusercontent.com/31706702/56012496-69c59000-5cb2-11e9-8e96-737523aeba7f.png">

**Comment by CarolinaVelasco** [comment]
@jcochanc @lindechen I just pushed the script variable_inclusion. Please look at the script. Consider the variation of the variables and additional variables we could add. I'll upload the codebooks shortly.
